### DISCUSSION TITLE
how can I connect to the http://photoprismpi.local/

### DISCUSSION THREAD

**Discussion by jxz2333** [photoprism/photoprism#4316] [Q&A]
I have run the img on my raspiberry,but when I use my computrer connect to the http://photoprismpi.local/,it's return 502.

**Comment by graciousgrey** [comment]
If you can't connect, try using the existing hostname or IP address instead.

In case this also does not work our troubleshooting checklist might help you: 

https://docs.photoprism.app/getting-started/troubleshooting/#connection-fails

**Reply by jxz2333** [reply]
My computer and Raspberry Pi are connected to the same WiFi. I can use SSH to connect to my Raspberry Pi address: 192.168.123.95. Can I use this IP address to connect to this service?

**Reply by lastzero** [reply]
Yes, that should be possible: http://192.168.123.95:2342/

**Reply by jxz2333** [reply]
This webpage still cannot be connected. I opened the firewall and used the netstat -tunlp command to check all open port numbers, but it seems that the project service is not running successfully. I also used dokcer ps to check the currently running image, but the result was empty. Is there any other way to check whether the project is working?
![image](https://github.com/photoprism/photoprism/assets/73346681/436ceea5-a03a-4146-92f0-7ae508d6fec7)

**Reply by lastzero** [reply]
How did you start the service in the first place? The `docker compose up` command displays logs that will tell you if the service has been started successfully. See our documentation for more troubleshooting tips:
- https://docs.photoprism.app/getting-started/troubleshooting/docker/#viewing-logs

**Reply by jxz2333** [reply]
The documentation says that you only need to burn this image, then turn on the computer, and the project will run automatically? https://docs.photoprism.app/getting-started/raspberry-pi/microsd-image/

**Reply by lastzero** [reply]
Maybe the Docker image is still downloading then? I can take a while, depending on your Internet connection speed.

**Reply by jxz2333** [reply]
Well, maybe you are right. Due to the Chinese firewall, the download speed may be lower.

**Reply by jxz2333** [reply]
Thank you for your answer！谢谢！
